src/client/epochs.rs

```diff
@@ -12,7 +12,7 @@ impl Maestro {
     }
 
     pub async fn specific_epoch(&self, epoch_no: i32) -> Result<EpochResp, Box<dyn Error>> {
-        let url = format!("/epochs/{}/info", epoch_no);
+        let url = format!("/epochs/{}", epoch_no);
```

Comment:
This endpoint was returning a 404 because the URL did not match the documented URL.